region/maridia/inner-pink.json

```diff
@@ -4895,33 +5012,68 @@
                   "requires": [
                     "Gravity",
                     "Morph",
+                    {"or": [
+                      "SpaceJump",
+                      "canCarefulJump",
+                      "h_canResetFallSpeed",
+                      "canSuitlessMaridia",
+                      "h_canHBJ",
+                      { "enemyDamage": { "enemy": "Cacatac", "hits": 1, "type": "spike" } }
+                    ]},
                     {"or":[
                       "SpaceJump",
                       "canWalljump",
                       "HiJump",
-                      "canCarefulJump",
+                      "h_canIBJ",
+                      "h_canHBJ",
+                      {"and": [
+                        "canSuitlessMaridia",
+                        "canCarefulJump"
+                      ]},
+                      "h_canSpringBallBombJump",
                       {"spikeHits": 1}
                     ]}
                   ],
-                  "note": "There is a way to avoid taking damage through the room, even without mid-air unmorphing."
+                  "note": "It is possible to jump over the spikes by disabling Gravity."
                 },
                 {
-                  "name": "Suitless",
+                  "name": "Suitless HiJump",
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
                     "Morph",
-                    {"or":[
-                      {"and":[
-                        "HiJump",
-                        "canWalljump"
-                      ]},
+                    "HiJump",
+                    {"or": [
+                      "canWalljump",
+                      "SpaceJump",
                       "canSpringBallJumpMidAir",
+                      {"and": [
+                        "h_canIBJ",
+                        "canBombJumpWaterEscape"
+                      ]},
+                      {"spikeHits": 1}
```

Comment:
or careful jump? then mention its easiest to jump over the spikes with hijump turned off